packages/webapp/src/components/FilterDropdown/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Markup from './Markup';
+
+class FilterDropdown extends Component {
+  constructor(props) {
+    super(props);
+    this.eventHandler = this.eventHandler.bind(this);
+
+    this.state = {
+      selected: '',
+    };
+
+    this.events = {
+      eventHandler: this.eventHandler.bind(this),
+    };
+  }
+
+  eventHandler(e) {
+    const { updateUrl, year } = this.props;
```

Comment:
Why do you have `updateUrl` and `year` in here? `FilterDropdown` should not know anything about the context outside of it.

Reply:
Removed